source/Camera/Basic/CaptureTutorial.md

```diff
@@ -0,0 +1,121 @@
+```cpp
+Zivid::CaptureAssistant::SuggestSettingsParameters suggestSettingsParameters(
+    std::chrono::milliseconds{ 1200 }, Zivid::CaptureAssistant::AmbientLightFrequency::none);
+
+std::cout << "Running Capture Assistant with parameters: " << suggestSettingsParameters << std::endl;
+const auto settingsVector{ Zivid::CaptureAssistant::suggestSettings(camera, suggestSettingsParameters) };
+```
+
+These settings can be used in an [HDR capture](#capture-hdr), which we will discuss later.
+
+As opposed to manual configuration of settings, there are only two parameters to consider with Capture Assistant.
+1. **Allowed budget for capture** in number of milliseconds.
+    1. Minimum allowed budget is 200ms. This allows only one frame to be captured.
+    2. The algorithm will combine multiple frames if the budget allows.
+    3. The algorithm will attempt to cover as much of the dynamic range in the scene as possible.
+    4. A budget of more than 1 second will get good coverage in most scenarios.
+2. **Ambient light compensation**
+    1. May restrict capture assistant to exposure periods that are multiples of the ambient light period.
+    2. 60Hz is found in (amongst others) Japan, Americas, Taiwan, South Korea and Philippines
+    3. 50Hz is found in most rest of the world
+
+### Manual configuration
+We may choose to configure settings manually ([go to source][settings-url]).
+```cpp
+camera << Zivid::Settings::Iris{ 20 } << Zivid::Settings::ExposureTime{ std::chrono::microseconds{ 8333 } }
+       << Zivid::Settings::Filters::Outlier::Enabled::yes << Zivid::Settings::Filters::Outlier::Threshold{ 5 };
+```
+
+## Part 3 - More on captures
+
+### Capture 2D
+If we only want to capture a 2D image, which is faster than 3D, we can do so via the 2D API ([go to source][capture2d-url]).
+```cpp
+auto frame = camera.capture2D(settings);
+```
+
+Note that settings are passed as argument. These must be initialized first, and optionally modified ([go to source][settings2d-url]).
+```cpp
+auto settings = Zivid::Settings2D();
+settings.set(Zivid::Settings2D::ExposureTime{ std::chrono::microseconds{ 10000 } });
+settings.set(Zivid::Settings2D::Gain{ 1.0 });
+settings.set(Zivid::Settings2D::Iris{ 35 });
+```
+
+### Capture HDR
+As was revealed in the [Capture Assistant](#capture-assistant) section, a capture may consist of multiple frames. In order to capture multiple frames, and combine them ([go to source][captureHDR-url])
+```cpp
+const auto hdrFrame{ Zivid::HDR::capture(camera, settingsVector) };
+```
+It is possible to manually create the `settingsVector`, if not set via [Capture Assistant](#capture-assistant).
+```cpp
+Add code here when sample has been updated
+```
+see issue https://github.com/zivid/cpp-extra-samples/issues/70
+
+
+## Bonus
+
+### Connect to a virtual camera [Optional]
```

Comment:
I think this should be above in the 'Capture' step. We want that people who don't have a camera to see this. If it is at the bottom it is not going to be seen straight away.